distributed/scheduler.py

```diff
@@ -5070,16 +5068,35 @@ def validate_state(self, allow_overlap: bool = False) -> None:
             raise ValueError("Workers not the same in all collections")
 
         assert self.running.issuperset(self.idle.values()), (
-            self.running,
-            list(self.idle.values()),
+            self.running.copy(),
+            set(self.idle.values()),
+        )
```

Comment:
sets were added by reference to the assertion, and by the time they were printed out they were empty!